third_party/docfx/templates/devsite/partials/uref/class.tmpl.partial

```diff
@@ -83,6 +83,105 @@
   </tbody>
 </table>
 {{/exceptions.0}}
+{{#arguments.0}}
+<div class="markdown level0 summary">
+  <strong>{{__global.parameter}}{{#arguments.1}}s{{/arguments.1}}</strong>
+  <ul>
+{{/arguments.0}}
+{{#arguments}}
+    <li>
+      {{{description}}}
+    </li>
+{{/arguments}}
+{{#arguments.0}}
+  </ul>
+</div>
+{{/arguments.0}}
+{{#overloads.0}}
+<div class="markdown level1 summary">
+  <strong>{{__global.overloads}}</strong>
+{{/overloads.0}}
+{{#overloads}}
+  <div class="codewrapper">
+    <pre><code class="prettyprint">{{content}}</code></pre>
+  </div>
+  <div class="markdown level1 summary">
+    {{{description}}}
+  </div>
+{{#arguments.0}}
+<div class="markdown level1 summary">
+  <strong>{{__global.parameter}}{{#arguments.1}}s{{/arguments.1}}</strong>
+  <ul>
+{{/arguments.0}}
+{{#arguments}}
+    <li>
+     {{{description}}}
+    </li>
+{{/arguments}}
+{{#arguments.0}}
+  </ul>
+</div>
+{{/arguments.0}}
+{{/overloads}}
+{{#overloads.0}}
+</div>
+{{/overloads.0}}
+{{#yields.0}}
+<div class="markdown level1 summary">
+  <strong>{{__global.yields}}</strong>
+  <ul>
+{{/yields.0}}
+{{#yields}}
+    <li>
+      {{{description}}}
+    </li>
+{{/yields}}
+{{#yields.0}}
+  </ul>
+</div>
+{{/yields.0}}
+{{#yieldparams.0}}
+<div class="markdown level1 summary">
+  <strong>{{__global.yieldparams}}{{#yieldparams.1}}s{{/yieldparams.1}}</strong>
+  <ul>
+{{/yieldparams.0}}
+{{#yieldparams}}
+    <li>
+      {{{description}}}
+    </li>
+{{/yieldparams}}
+{{#yieldparams.0}}
+  </ul>
+</div>
+{{/yieldparams.0}}
+{{#returnValues.0}}

```

Comment:
There is also a returns section below. Let's try to consolidate for the same reason.

Reply:
Due to https://github.com/googleapis/doc-templates/issues/250 combined with the large difference in information organization between what rubyists would expect and what the current return/parameters tables output, I and the other rubyists I talked to wanted to recreate some familiarity. 

The table doesn't work for us for parameters, and it wouldn't really make sense for the "yields" or "raises" field. People would expect them to be formatted similarly. Using the already written table would be a departure of style from elsewhere in our docs.

https://cloud.devsite.corp.google.com/ruby/docs/reference/Google-Cloud-Vision-V1/latest/Google-Cloud-Vision-V1-ImageAnnotator-Client#Google__Cloud__Vision__V1__ImageAnnotator__Client_crop_hints_detection_instance_ and https://googleapis.dev/ruby/google-cloud-vision-v1/latest/Google/Cloud/Vision/V1/ImageAnnotator/Client.html#crop_hints_detection-instance_method for context

Reply:
Perhaps we shouldn't have a table at all?

Reply:
We discussed this offline and it's a bigger change because we'd have to possibly update every table to a list to keep consistency. So, sounds good to keep that as-is for now.